gui/src/components/Panel.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+  import IconButton from './IconButton.svelte';
+  import BackSVG from './mono/leftarrow.svelte';
+  import * as router from 'svelte-spa-router';
+
+  export let title: string | undefined = undefined;
+  export let backRoute: string | undefined = undefined;
+</script>
+
+<div class="panel">
+  {#if $$slots.actions || title !== undefined || backRoute !== undefined}
+    <header>
+      <div class="header-title">
+        {#if backRoute !== undefined}
```

Comment:
Again, in template code, really convenient to just write `{#if backRoute}` even tho it's less explicit, it's basically _why_ JavaScript has this sort of truthiness checks in the first place.